DOC-2268: Add new documentation for `ExportWord` Plugin. (#3072)
tinymce/tinymce-docs: modules/ROOT/partials/commands/exportword-cmds.adoc

[cols="1,3",options="header"]
|===
|Command         |Description

> |exportWord   |Generates word file from calling `editor.getContent()` and exports it to new file `document.docx`.

Reviewer: ```suggestion
|ExportWord   |Generates word file from calling `editor.getContent()` and exports it to new file `document.docx`.
```

> |ExportWord   |Generates word file from calling `editor.getContent()` and exports it to new file `document.docx`.

Reviewer 2: ```suggestion
|ExportWord   |Uses `editor.getContent()` and generates a new `document.docx` Word file that can be saved to local disk.
```

|===

.Examples
[source,js]
----

> tinymce.activeEditor.execCommand('exportWord');

Reviewer: ```suggestion
tinymce.activeEditor.execCommand('ExportWord');
```
